.husky/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
Does this work on Windows?

Reply:
I tested this in a variety of environments with the official Windows Git packages installed via the Chocolately package manager (as recommended by a certified Windows developer contact), and the script itself seems to work. I did commit a small configuration fix for Prettier, setting `endOfLine` to `auto` in order to work well with Git's automatic line handling settings.